concrete/blocks/top_navigation_bar/controller.php

```diff
@@ -306,5 +306,19 @@ public function getPageItemNavTarget($pageItem) // Respect nav_target Page Attri
 	
 	return $target;
     }
+    
+    public function getPageItemURL($pageItem) 
```

Comment:
This is interesting but I don't think it's the place for this. We have that PageItem navigation class - that's where we ought to be checking whether a page is an external link, and if it is, swapping out the getCollectionLink method with the getCollectionPointerExternalLink. That will fix this for all other places that use the page navigation classes.

Reply:
@aembler before I dive in, \concrete\core\navigation\item\item getURL() is where I'm looking to make the changes then correct?

Reply:
I understand and have this working on my end.  Will create new PR and close this one in favor at that time.  Most likely tomorrow.